.gitignore

```diff
@@ -19,6 +19,14 @@ DerivedData
 *.hmap
 *.ipa
 
+# Swift Package Manager
+Package.resolved
```

Comment:
Don't exclude `Package.resolved` and `Podfile.lock` as they're used to constrain specific version. Also, bring back `Podfile.lock` file and add `Package.resolved` to the tracked files.